sros/docker/launch.py

```diff
@@ -773,10 +821,56 @@ def __init__(self, hostname, username, password, mode, variant_name, conn_mode):
                     major_release,
                     variant,
                     conn_mode,
-                ),
-                SROS_lc(variant, conn_mode, variant["max_nics"]),
+                )
             ]
 
+            # LC VM Instantiation
+            start_eth = 1
+            lc_slot_tracker = []
+            for lc in variant["lcs"]:
+                lc_slot = lc["slot"]
+
+                # Validation of lc_slot value
+                if not lc_slot:
+                    self.logger.warning(
+                        f"No Slot information on following lc line defintion: {lc}"
+                        "Skip LC VM creation"
+                    )
+                    continue
+
+                if lc_slot in lc_slot_tracker:
+                    self.logger.warning(
+                        f"Found duplicate slot: {lc} Skip LC VM creation"
+                    )
+                    continue
+
+                try:
+                    lc_slot = int(lc_slot)
+                except (TypeError, ValueError):
+                    self.logger.warning(
+                        f"slot value format is not valid: {lc} Skip LC VM creation"
+                    )
+                    continue
+
+                # Priority is to use max_nics from each lc definition
+                max_nics = lc.get("max_nics", None)
+                if not max_nics:
+                    max_nics = variant["max_nics"]
+
+                self.vms.append(
+                    SROS_lc(
+                        lc,
+                        conn_mode,
+                        max_nics,
+                        slot=lc_slot,
```

Comment:
Hi @gusman 
i have changed this line from 

```
slot=2+lc_slot
```

to
```
slot=lc_slot
```

because I believe slot number is a unique integer provided by the user, therefore it won't interfere with the overlay disk image name.
Do you agree?

Reply:
I agree.